### PR TITLE
1-game more challenging,  2- more levels fixes #24 #57

### LINKED ISSUE #24: Increment amount of levels

### PR DESCRIPTION
**Pull request by yoav1994**
1) 1500 levels instead of 10
2) as the level increase, the game becomes more challenging (more enemies, in bigger waves flows)

fixes #24 #57

### CODE REVIEW

**Review by Coder-DG — CHANGES_REQUESTED** [review]
You've overridden the `circle.yml` file. You probably resolved the conflict by replacing the file to your own version or maybe something else happened.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (Coder-DG) [inline LICENSE:1]: License already existed, are you sure this is a necessary change?

**Review by Coder-DG — CHANGES_REQUESTED** [review]

**Comment by Coder-DG** [comment]
I think you need to rebase to master, not sure

**Comment by davidginzbourg** [comment]
fixes #24 #57

**Pull request closed without merging**